### DISCUSSION TITLE
Asciidoc extension

### DISCUSSION THREAD

**Discussion by chuck-confluent** [community/community#7925] [Codespaces] [answered]
Hey y'all, I love Github.dev. Once thing -- the Asciidoc extension does not currently work. I asked over on the extension repo:
- https://github.com/asciidoctor/asciidoctor-vscode/issues/464

The extension appears to work fine in gitpod, but gitpod is a little heavy for much of the asciidoc work I want to do. I don't actually need a workspace. I just need a quick text editor (Github.dev is perfect for this).

They are having issues getting the extension to work in github.dev:
- https://github.com/asciidoctor/asciidoctor-vscode/issues/429

**Comment by bamurtaugh** [comment, marked as answer]
Hi @chuck-confluent, thanks for opening, and so glad you're loving github.dev!

There are some steps an extension can take to ensure it works in the web: https://code.visualstudio.com/api/extension-guides/web-extensions.

It looks like the Asciidoc team is aware of this and has this itemized and on their radar: https://github.com/asciidoctor/asciidoctor-vscode/issues/429#issuecomment-950136729 and in PR [here](https://github.com/asciidoctor/asciidoctor-vscode/pull/473).

Please let me know if you have any other questions or feedback!
